### DISCUSSION TITLE
Rooms that won't start because the scene file won't load

### DISCUSSION THREAD

**Discussion by rawnsley** [mozilla/hubs#2807] [General Questions & Discussion] [answered]
I have killed a few rooms by uploading GLB scene files that are either malformed or contain elements that Hubs cannot process. For instance, the navigation mesh is too large. When this happens the room will not load enough to allow access to any of the UI that allows you to change scene. It can still be done through the browser developer console:

1. Load the broken room
2. Open the developer console
3. Enter the following: `document.getElementById("avatar-rig").messageDispatch.hubChannel.updateScene("https://hubs.mozilla.com/scenes/CzmA8a0/hello-webxr-main-hall")`
4. Refresh the browser

**Comment by emclaren** [comment, marked as answer]
Hi rawnsley - if I'm reading this correctly this is just a tip and not a question. Thanks for sharing - we've had folks ask about this before and I'm happy to have something to send when this happens. 🙌 High Five!
